stdlib/2/httplib.pyi

```diff
@@ -36,6 +36,7 @@ class HTTPResponse:
     def fileno(self): ...
     def getheader(self, name, default=None): ...
     def getheaders(self): ...
+    def begin(self) -> None: ...
```

Comment:
This method does actually already exist in the Python 2 stub, a few lines above, although it is not annotated. This duplication is also what makes CI fail.